corehq/apps/app_manager/views/formdesigner.py

```diff
@@ -194,3 +194,8 @@ def get_data_schema(request, domain, app_id=None, form_unique_id=None):
     if "pretty" in request.GET:
         kw["indent"] = 2
     return HttpResponse(json.dumps(data, **kw))
+
+
+@require_GET
```

Comment:
yeah looked at [the middleware](https://github.com/dimagi/commcare-hq/blob/master/corehq/middleware.py#L116-L140) and I think this should be fine. if the user's not logged in it will be a noop and if they are logged in it should extend the session